assets/tests/components/ladderPage.test.tsx

```diff
@@ -199,6 +201,7 @@ describe("LadderPage", () => {
   })
 
   test("can save a route tab as a preset from the save icon", async () => {
+    mockFullStoryEvent()
```

Comment:
question: Would we want this mocked for all tests? Maybe worth pulling into `setup.tsx`

Reply:
I like the declarative approach I've been taking by only doing it per test that needs it, and nothing should fail because all `FS.event` calls use typescript's null operator `window.FS?.event`.
But I'm open to putting it into `setup.tsx`